app/models/event.rb

```diff
@@ -14,11 +14,12 @@ def self.active(early_access: false)
 
   def low_income_open?
     are_missing = !low_income_requests_start.present? && !low_income_requests_end.present?
-    outside_window = Time.zone.now.before?(low_income_requests_start) || Time.zone.now.after?(low_income_requests_end)
-    if are_missing || outside_window
+    if are_missing
       false
     else
-      event_mode == 'prerelease'
+      inside_window = !Time.zone.now.before?(low_income_requests_start) &&
+                      !Time.zone.now.after?(low_income_requests_end)
+      inside_window && event_mode == 'prerelease'
```

Comment:
why did this change? I found it easier to read before personally.

Reply:
Because if it was ever `nil` then trying to do `Time.zone.now.before?` / `after?` on a nil object threw an exception, this avoids that

Reply:
Could we not have the bangs then? So `Time.zone.now.before?(low_income_requests_end)` instead of `!Time.zone.now.before?(low_income_requests_start)`?